state-sync/src/request_manager.rs

```diff
@@ -78,38 +79,38 @@ enum PeerScoreUpdateType {
 }
 
 pub struct RequestManager {
-    // list of peers that are eligible for this node to send sync requests to
-    // grouped by network preference
-    eligible_peers: BTreeMap<usize, (Vec<PeerNetworkId>, Option<WeightedIndex<f64>>)>,
+    // The list of peers that are eligible for this node to send sync requests
+    // to (grouped by network).
+    eligible_peers: BTreeMap<NetworkId, (Vec<PeerNetworkId>, Option<WeightedIndex<f64>>)>,
```

Comment:
Can you elaborate here? 

My understanding is that `NodeNetworkId` is only required for supporting multiple public networks (i.e., pfns). By only considering a peer's `NetworkId` as opposed to `NodeNetworkId`, state sync treats all nodes across different pfns as potential state sync peers in the same layer (regardless of what pfn the peer is in). This is to say, if state sync needs to select a node from a pfn to send a chunk request to, it'll disregard what network the pfn comes from. I think this is acceptable behaviour. In fact, I'm not sure we'd want behaviour that is different here (e.g., if you have multiple pfns, how would you define an importance ordering over them if you wanted to prefer one vs. the other?)

Reply:
Oh, I thought it needed to be (because I wasn't sure how long it would be until i got rid of the 2nd Public network interface).  But, I take that back now I figured out how to :)

Reply:
Cool, thanks @gregnazario! Sounds good, we should be good here already then 😄